azure-pipelines.yml

```diff
@@ -50,7 +50,8 @@ stages:
     - script: |
         set -ex
         sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
```

Comment:
>libhiredis-dev [](http://example.com/codeflow?start=30&length=14)

The intention of purging is to remove all the self-build packages from the env. Since it is not self-build anymore, will this line fail? do you want to stop purge this package?

Reply:
No, I want keep this line no change, only install missing libhiredis0.14.
Also the next line, does not install self-build libhiredis-dev:

sudo dpkg -i ../target/debs/buster/{libyang_1.0.73_amd64.deb,libswsscommon_1.0.0_amd64.deb,python3-swsscommon_1.0.0_amd64.deb,libnl-3-200_*.deb,libnl-genl-3-200_*.deb,libnl-nf-3-200_*.deb,libnl-route-3-200_*.deb}